common/txmgr/types/tx_store.go

```diff
@@ -79,6 +79,8 @@ type TransactionStore[
 	// Search for Tx using the fromAddress and sequence
 	FindTxWithSequence(ctx context.Context, fromAddress ADDR, seq SEQ) (etx *Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE], err error)
 	FindNextUnstartedTransactionFromAddress(ctx context.Context, fromAddress ADDR, chainID CHAIN_ID) (*Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE], error)
+
+	// FindTransactionsConfirmedInBlockRange retrieves tx with attempts which contain attempts and partial receipts values for optimization purpose
```

Comment:
The wording here seems a little redundant
```suggestion
	// FindTransactionsConfirmedInBlockRange retrieves tx with attempts which contain partial receipts values for optimization purpose
```